src/libraries/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -6,6 +6,8 @@
     <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
     <Nullable>enable</Nullable>
     <IsPackable>true</IsPackable>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
FWIW the versioning info [here](https://github.com/dotnet/runtime/blob/4684a1dbab50ea759eaf970302fc9c1f7007bbc2/docs/project/library-servicing.md)